cmd/dep/graphviz.go

```diff
@@ -91,3 +91,24 @@ func (dp gvnode) label() string {
 
 	return strings.Join(label, "\n")
 }
+
+// Ensure that the literal string prefix is a path tree match and
```

Comment:
Even though this isn't exported, it should follow the Go standard of having the comment block begin with the func name.